### PR TITLE
Adds medical tape, a worse version of splints

### PR DESCRIPTION
**Pull request by QualityVan**
## About The Pull Request
Basically the equivalent of gauze/ointment compared to advanced kits. Easy to use, no penalty for low medical skill, but not as good. Will fall off in 5 minutes.
Sprite provided by kuro, thanks.
Also makes regular splints slightly sturdier when applied without any skill, but take longer to use. Trained use time and sturdiness aren't changed.

## Why It's Good For The Game
Good to have worse but low-skill medical alternatives.
I'd also like to make these infinite from vendors later, along with gauze and ointment.

## Changelog
:cl:
add: Medical vendors now have tape, which can be used as a worse but untrained alternative to splints.
balance: Splints applied by untrained people are sturdier, but take a little longer to use.
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/medical.dm [hunk 1]
```diff
@@ -298,13 +303,11 @@
 			to_chat(user, span_warning("You can't apply a splint there!"))
 			return
 
-		if(affecting.limb_status & LIMB_DESTROYED)
-			to_chat(user, span_warning("[user == M ? "You don't" : "[M] doesn't"] have \a [limb]!"))
-			return
-
-		if(affecting.limb_status & LIMB_SPLINTED)
-			to_chat(user, span_warning("[user == M ? "Your" : "[M]'s"] [limb] is already splinted!"))
-			return
```
Comment (QualityVan) [inline code/game/objects/items/stacks/medical.dm:307]: These checks aren't necessary because the limb itself handles them in its apply_splints

**Comment by Losenis** [comment]
>A worse version of something that already turns a tyr2 into a 1hit kill by organ damage
I'd sooner retexture normal splints with this and make splints only fall off when they're in bonefrac territory, but lock it to corpsmen skill and up for usage.

File: code/game/objects/items/stacks/medical.dm (hunk 1 above)
Comment (BraveMole) [reply code/game/objects/items/stacks/medical.dm:307]: does that mean you have the windup then the warning?

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/game/objects/items/stacks/medical.dm (hunk 1 above)
Comment (QualityVan) [reply code/game/objects/items/stacks/medical.dm:307]: You'll have the fumble delay first, then the cancel out if the limb is destroyed/splinted, and then the splinting do_after.

Comment (BraveMole) [reply code/game/objects/items/stacks/medical.dm:307]: so keeping the checks would be better no? why would you fumble to check if a splint is still here or destroy?

Comment (QualityVan) [reply code/game/objects/items/stacks/medical.dm:307]: Gross incompetence. You can also avoid this by looking at the person, which tells you if any limbs are splinted or missing.

**Review by Kuro020 — COMMENTED** [review]
image
:mistake: this sprite is just no

**Review by Kuro020 — CHANGES_REQUESTED** [review]
see above ^

**Comment by Kuro020** [comment]
tape
use this one

**Review by Kuro020 — APPROVED** [review]

**Comment by lbnesquik** [comment]
I don't like the idea of the thing falling off. At least, not without some really clear indication that this thing is on a timer. 
Maybe large font text?
You could claim it is a skill issue if someone doesn't notice, but if someone else patches you up, and you don't check, your fracture comes right back.

**Comment by QualityVan** [comment]
> 
> 
> I don't like the idea of the thing falling off. At least, not without some really clear indication that this thing is on a timer.
> Maybe large font text?
> You could claim it is a skill issue if someone doesn't notice, but if someone else patches you up, and you don't check, your fracture comes right back.
you know what you right, I'll add a warning a couple minutes before it falls off

**Comment by SplinterGP** [comment]
could maybe instead of falling off or something just give a bit more debuffs than the splints do?

**Comment by QualityVan** [comment]
> 
> 
> could maybe instead of falling off or something just give a bit more debuffs than the splints do?

I've thought about this a bit, and splints really don't have any debuffs as is. There's being easier to break than an unbroken limb, but tape is already worse there. Making tape have finite duration also plays into the idea that splinting is inherently a temporary measure, needing either reapplication later or more permanent treatment.

**Comment by QualityVan** [comment]
Probably too strong as is and also just not interesting.

**Pull request closed without merging**